openbb_platform/core/openbb_core/app/model/field.py

```diff
@@ -0,0 +1,27 @@
+"""Custom field for OpenBB."""
+
+from typing import Any, List, Optional
+
+from pydantic.fields import FieldInfo
+
+
+class OpenBBField(FieldInfo):
+    """Custom field for OpenBB."""
+
+    def __repr__(self):
+        """Override FieldInfo __repr__."""
+        # We use repr() to avoid decoding special characters like \n
+        if self.choices:
+            return f"OpenBBField(description={repr(self.description)}, choices={repr(self.choices)})"
+        return f"OpenBBField(description={repr(self.description)})"
+
+    def __init__(self, description: str, choices: Optional[List[Any]] = None):
+        """Initialize OpenBBField."""
+        super().__init__(
+            description=description, json_schema_extra={"choices": choices}
+        )
```

Comment:
This will force `choices=None` when no choices are provided, which pollutes the resulting pydantic model schema of the function signature. Instead, we should do the following: 
```suggestion
        """Initialize OpenBBField."""
        json_schema_extra = {"choices": choices} if choices else None
        super().__init__(
            description=description, json_schema_extra=json_schema_extra
        )
```

cc @montezdesousa